docs/change-log.md

```diff
@@ -1,6 +1,27 @@
 Changelog
 ==========
 
+7.1.0
+-----
+### Upgrade Notes
+- Bumped minimum engine API version to 1.24
+- Bumped default engine API version to 1.44 (Moby 25.0)
```

Comment:
If #3261 is planned to get merged for 7.1.0, this needs to be updated.

Reply:
We were discussing that one, and maybe (given that there may still be some questions remaining there), we should skip that one for now, and leave it for the next release.